pyproject.toml

```diff
@@ -26,7 +24,9 @@ testpaths = [
 [project.optional-dependencies]
 dev = [
     "pytest",
-    "twine"
+    "twine",
+    "flake8",
```

Comment:
As part of this PR, lets add flake8 and pyright checks in the CI

Reply:
Integrated both flake8 and pyright in the CI.

Reply:
Thanks!